proposals/simd/SIMD.md

```diff
@@ -675,3 +684,28 @@ Lane-wise saturating conversion from floating point to integer using the IEEE
 resulting lane is 0. If the rounded integer value of a lane is outside the
 range of the destination type, the result is saturated to the nearest
 representable integer value.
+
+
+## Reductions
+
+There is no instruction for reductions.
+Instead, one can use permutations to reduce lane-wise operations like `add`, `min`, `max`, `and`, `or`...
+
+Here is an example to reduce add on f32x4:
+```
+get_local 0
+v64x2.permute 1 0  ;; swap the lower part with the higher part of the vector
```

Comment:
Nit: `permute` is not a part of the proposal anymore, you might want to use `shuffle_var`.

Reply:
Fixed